### PR TITLE
add auto plugin versioning

### PR DESCRIPTION
**Pull request by dm432**

### CODE REVIEW

**Review by ChechuDEV — DISMISSED** [review]
The use of the count can be quite redundant imo, but that's the approved versioning, so all good.

**Review by ChechuDEV — DISMISSED** [review]
All good. Didn't interpret a file good enough.

File: build.gradle.kts [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+            it.bufferedReader().use(BufferedReader::readText)
+        }
+        process.destroy()
+        output.trim()
+    }
+
+// example version number: MPP-0.1.302-1.19.4-SNAPSHOT
+// when changing version be sure to only use the chars mentioned in the comments
+// otherwise you have to change the regex in the workflows
+fun getVersionNumber(): String {
```
Comment (ChechuDEV) [inline build.gradle.kts:22]:
```suggestion
fun getPluginVersion(): String {
```

**Review by ChechuDEV — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (dm432) [reply build.gradle.kts]: Done

**Review by ChechuDEV — APPROVED** [review]
All good :D

**Pull request merged**